sponsorship/src/test/java/com/kyriba/conference/sponsorship/api/SponsorControllerApiTest.java

```diff
@@ -48,7 +46,7 @@ void setUp(RestDocumentationContextProvider restDocumentationContextProvider)
   void registerSponsor()
   {
     Number id = given(specification)
-        .contentType(MediaType.APPLICATION_JSON_UTF8_VALUE)

```

Comment:
what is the issue with 'APPLICATION_JSON_UTF8_VALUE'?

Reply:
Deprecated as of Spring Framework 5.2 in favor of APPLICATION_JSON_VALUE since major browsers like Chrome now comply with the specification  and interpret correctly UTF-8 special characters without requiring a charset=UTF-8 parameter.